book/website/reproducible-research/reviewing/reviewing-workflow.md

```diff
@@ -2,44 +2,65 @@
 
-Within the Github environment, projects can be configured to *require* a review before a given pull request can be merged. Even if this option hasn't been selected, it's still possible (and indeed best practice) to manually request a review on a pending PR.
+This section focuses on the typical workflows behind a formal review process, as commonly implemented within a social coding environment like Github.
 
 <a name="prepare_the_code"></a>
 ### Prepare the code
 
-Before requesting a review, be sure you've met all the obvious quality benchmarks for the project you are contributing to. This means making sure:
+Before requesting a review, make sure you've met all the obvious quality benchmarks for the project you are contributing to. This means making sure you have check the review list (see {ref}`checklist for the coder<rr-checklist-for-code-review>`).
+
+A reviewer should check these things (see {ref}`checklist for the coder<rr-checklist-for-code-review>`), but defects on these fronts should be by occasional oversight, rather than systematic.
+
+<a name="forks_and_branches"></a>
+### Propose changes
 
-- you have created [documentation](#Documentation) to the required standards of the project,
-- you have [tested](#Improvements_to_testing) your code to the required standards of the project,
-- your code is not causing the tests in the main project to fail (many [continuous integration](/continuous_integration/continuous_integration) systems will test this automatically for you once you create the PR), and
-- you believe your code meets the declared [style guide](../../code_quality/code_quality#code-style) for the project.
+In the GitHub system, the review is begun directly from and often accessed through the [pull request page](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
+The review step occurs between the points where the coder believes their contribution is complete and where that contribution is merged into the trunk code for the project, and so it is intimately associated with a single pull request.
 
-A reviewer should check these things, but defects on these fronts should be by occasional oversight, rather than systematic.
+Within the Github environment, projects can be configured to *require* a review before a given pull request can be merged.
+Even if this option hasn't been selected, it's still possible (and indeed best practice) to manually request a review on a pending pull request.
 
 <a name="create_discuss_change"></a>
-### Create and discuss the review; make the changes
+### Create and discuss the review
 
-At this point, the review process can begin. In Github, the reviewer can provide both general comments as well as line-by-line comments.
+At this point, the review process can begin. In Github, the reviewer can provide both general comments as well as line-by-line comments, see [GitHub code review](https://github.com/features/code-review).
```

Comment:
Other open code repositories like BitBucket or GitLab could have features similar to the described ones in all these texts. If it is not already written, there should be a brief sentence telling there could be conceptually similar mechanisms in other systems, but they are not explained here.

Reply:
Agree. A sentence telling about other systems is added to the second paragraph in section **Formal vs informal reviews.**